live_validation_scripts/asana_reports.mjs

```diff
@@ -258,18 +326,33 @@ async function main() {
             } catch (attachmentError) {
                 console.error(`Error adding attachment for ${dirPath}:`, attachmentError.message);
                 console.error('Full error:', attachmentError);
+                hasErrors = true;
             }
 
+            // Even if there are no errors, continue to create per-owner subtasks where possible
             if (MAKE_PER_OWNER_SUBTASKS) {
                 // Create subtasks for each owner
                 for (const owner of ownersWithErrors) {
-                    await createOwnerSubtask(owner, taskGid);
+                    const success = await createOwnerSubtask(owner, taskGid);
+                    if (!success) {
+                        console.error(`Error creating subtask for ${owner}`);
+                        hasErrors = true;
+                    }
                 }
             }
         }
     } catch (error) {
         console.error(`Error creating task for ${dirPath}:`, error);
+        process.exit(1);
```

Comment:
If we're exiting with error here, might as well just avoid the try-catch altogether